internal/workload/podspec_updates_test.go

```diff
@@ -643,11 +645,55 @@ func TestProxyCLIArgs(t *testing.T) {
 			wantWorkloadEnv: map[string]string{
 				"CSQL_PROXY_SQLADMIN_API_ENDPOINT": "https://example.com",
 				"CSQL_PROXY_HTTP_PORT":             "9092",
+				"CSQL_PROXY_ADMIN_PORT":            "9091",
+				"CSQL_PROXY_DEBUG":                 "true",
 				"CSQL_PROXY_HEALTH_CHECK":          "true",
 				"CSQL_PROXY_MAX_CONNECTIONS":       "10",
 				"CSQL_PROXY_MAX_SIGTERM_DELAY":     "20",
 			},
 		},
+		{
+			desc: "debug flag ignored enabled when admin port is not set",
+			proxySpec: v1alpha1.AuthProxyWorkloadSpec{
+				AuthProxyContainer: &v1alpha1.AuthProxyContainerSpec{
+					Telemetry: &v1alpha1.TelemetrySpec{
+						Debug: ptr(true),
```

Comment:
Should this be an error?

Reply:
Hmm. This gets caught by AuthProxyWorkload.ValidateCreate(). I've updated the test to call ValidateCreate() to ensure all the test cases are valid before attempting to configure the proxy. Also I removed this test case since it was only relevant for /quitquitquit